FizzBuzz.cs

```diff
@@ -0,0 +1,23 @@
+public class Solution
+{
+    private bool IsDivisibleBy (int dividend, int divisor)
+    {
+        return dividend % divisor == 0;
+    }
+    public IList<string> FizzBuzz(int n)
+    {
+        List<string> lista = new List<string>();
+        for (int i = 1; i <= n; i++)
+        {
+            string result= i switch
```

Comment:
I just see one more thing visually. Formatting should be something really automatic, have you tried pressing these buttons? I mention these small formatting issues as normally a repository would not let you merge code with something like `result= i` instead of `result = i`

<img width="774" height="262" alt="Screenshot 2025-08-19 at 8 37 50 PM" src="https://github.com/user-attachments/assets/427c21de-f70d-40d1-885a-8a323ac47d90" />


```suggestion
    }
    
    public IList<string> FizzBuzz(int n)
    {
        List<string> lista = new List<string>();
        for (int i = 1; i <= n; i++)
        {
            string result = i switch
```